src/content/docs/en/reference/image-service-reference.mdx

```diff
@@ -3,50 +3,108 @@ title: Image Service API
 i18nReady: false
 ---
 
-Astro [Assets](/en/guides/assets/) were designed to make it easy for any image optimization service to build a service on top of Astro.
+Astro [assets](/en/guides/assets/) were designed to make it easy for any image optimization service to build a service on top of Astro.
 
-::::caution
+:::caution
 The Image Services API is an _experimental_ Astro feature introduced in v2.1. Be mindful that the API is subject to change before it is marked as stable.
 :::
 
-Two types of services exists: Local and External.
+## What is an Image Service?
 
-- Local services handle the image transformation directly at build in SSG / runtime in dev / SSR. You can think of those as wrapper around libraries like Sharp, ImageMagick or Squoosh.
-- External services point to URLs and can be used for adding support for services such as Cloudinary, Vercel or any RIAPI-compliant server.
+Astro provides two types of image services: Local and External.
 
-Services definitions take the shape of an exported default object with various required methods ("hooks") used to create all the required properties. The major difference, API-wise, between Local and External services is the presence of a `transform` method doing the actual transformation.
+- **Local services** handle image transformations directly at build for static sites, or at runtime in dev / SSR. These are wrappers around libraries like Sharp, ImageMagick or Squoosh.
```

Comment:
I'd add a bit of nuance here, as it is possible (and probable) for them to be something else

```suggestion
- **Local services** handle image transformations directly at build for static sites, or at runtime in dev / SSR. These are often wrappers around libraries like Sharp, ImageMagick or Squoosh.
```